l5kit/l5kit/rasterization/rasterizer_builder.py

```diff
@@ -148,21 +148,19 @@ def build_rasterizer(cfg: dict, data_manager: DataManager) -> Rasterizer:
             dataset_meta = _load_metadata(dataset_meta_key, data_manager)
             pose_to_ecef = np.array(dataset_meta["pose_to_ecef"], dtype=np.float64)
         except (KeyError, FileNotFoundError):  # TODO remove when new dataset version is available
-            print(
-                "!!dataset metafile not found!! the hard-coded matrix will be loaded.\n"
-                "This will be deprecated in future releases"
-            )
-            pose_to_ecef = np.asarray(
-                [
-                    [8.46617444e-01, 3.23463078e-01, -4.22623402e-01, -2.69876744e06],
-                    [-5.32201938e-01, 5.14559352e-01, -6.72301845e-01, -4.29315158e06],
-                    [-3.05311332e-16, 7.94103464e-01, 6.07782600e-01, 3.85516476e06],
-                    [0.00000000e00, 0.00000000e00, 0.00000000e00, 1.00000000e00],
-                ],
-                dtype=np.float64,
-            )
+            pose_to_ecef = get_hardcoded_pose_to_ecef()
 
         map_to_sat = np.matmul(ecef_to_sat, pose_to_ecef)
         return SatelliteRasterizer(raster_size, pixel_size, ego_center, sat_image, map_to_sat)
+    elif map_type == "semantic_debug":
```

Comment:
What do you think of managing this in the py_semantic branch, as only the line containing the class instantiation is different?
Note scope of the PR, but also holds for satellite map.

Reply:
yeah I don't see why not :) I'll do it for both here